Lib/test/support/__init__.py

```diff
@@ -1160,6 +1160,12 @@ def refcount_test(test):
     return no_tracing(cpython_only(test))
 
 
+def requires_test_modules(test):
```

Comment:
I'm not sure that it's useful since there is no change using it yet. Do you plan to use it?

Reply:
I agree. Let's remove it for now.

Reply:
Removed in b8af54a78bf55720f90eed6a589b63e67379b67b.